app/assets/javascripts/initializers/initializeCommentDropdown.js

```diff
@@ -102,7 +106,9 @@ function initializeCommentDropdown() {
         )[0];
 
         document.addEventListener('click', outsideClickListener);
-        clipboardCopyElement.addEventListener('click', iOSCopyText);
+        if (document.body.contains(clipboardCopyElement)) {
+          clipboardCopyElement.addEventListener('click', iOSCopyText);          
+        }
```

Comment:
same as the initial comment, what there is to gain on using contains since the object has already been queried with `getElementsByTagName` ?